hack/vgopath-setup.sh

```diff
@@ -21,3 +22,4 @@ TARGET_DIR="${REPO_ROOT:-$SCRIPT_DIR/..}"
 
 export GOROOT="${GOROOT:-"$(go env GOROOT)"}"
 export GOPATH="$VIRTUAL_GOPATH"
+export PATH="$PATH:$GOROOT/bin:$GOPATH/bin"
```

Comment:
The existing `PATH` could shadow binaries in `$GOROOT/bin` and `$GOPATH/bin`. Should we switch the order?
```suggestion
export PATH="$GOROOT/bin:$GOPATH/bin:$PATH"
```